server/views/pages/managementReporting/index.njk

```diff
@@ -30,15 +30,17 @@
   <p>Below is the of available management reports</p>
 
   {% for definition in definitions %}
+    {% if definition.authorised %}
```

Comment:
Check if the user is authorised to view report - at the moment this will always return true